setup.py

```diff
@@ -90,10 +90,12 @@
         'requests>=2.25.1,<3.0',
```

Comment:
If we're feeling it is too risky to uncap `requests` (whose latest release is [`2.27.1`](https://pypi.org/project/requests/)), then let's add a TODO comment to revisit early in the next sprint.

Reply:
unpinned